app/helpers/content_helper.rb

```diff
@@ -91,6 +92,12 @@ def markdown_youtube(text)
     end
   end
 
+  def markdown_video(text)
+    text.gsub /\[video\s+(.*?)\]/ do
```

Comment:
This matches "[video ]"

Reply:
I ended up changing it to `/\[video\s(https?:\/\/\S+)\]/`, this ensures it's a url as well since [video abc] would have resulted in an error.